lib/controller.js

```diff
@@ -68,7 +72,7 @@ Tessel.list = function(opts) {
             // Report that selected Tessel to the user
             logs.info('Multiple Tessels found.');
             if (tessel) {
-              logs.info('Will default to', tessel.name, '.');
+              logs.info('Will default to', tessel.name + '.');
```

Comment:
This was bugging me because it was print out as: `Will default to Frank .` with a space before the period.